doc/stdlib/arrays.rst

```diff
@@ -1045,7 +1051,7 @@ dense counterparts. The following functions are specific to sparse arrays.
    For an in-place version and algorithmic information, see :func:`Base.SparseArrays.dropzeros!`\ .
 
 .. function:: permute{Tv,Ti,Tp<:Integer,Tq<:Integer}(A::SparseMatrixCSC{Tv,Ti}, p::AbstractVector{Tp},
-..............    q::AbstractVector{Tq})
+                  q::AbstractVector{Tq})
```

Comment:
Misleading more than mistaken. "Wildcard" should probably be replaced with "single space", since, although `genstdlib` can deal with wildcards, Sphinx doesn't seem to like them and requires spaces.